### PR TITLE
Mainly UI Updates

### PR DESCRIPTION
**Pull request by OzGav**

### CODE REVIEW

File: docs/faq/troubleshooting.md [hunk 1]
```diff
@@ -33,6 +33,12 @@ Before you raise an issue [read this first](../support.md). Report issues using
 !!! note
     You can retrieve the full MA logs by going to the MA settings and clicking on CORE
 
+# A provider isn't working
+
+Naviagte to MA settings and inspect the provider entry. If there is a red circle with an exclamation mark next to it then hover over that icon to see the error message.
```
Comment (marcelveldt) [inline docs/faq/troubleshooting.md:38]:
```suggestion
Navigate to MA settings and inspect the provider entry. If there is a red circle with an exclamation mark next to it then hover over that icon to see the error message.
```

File: docs/music-providers/filesystem.md [hunk 2]
```diff
@@ -34,7 +34,9 @@ Music Assistant has support for SMB (also known as samba or CIFS) shares and DFS
 - Use the following naming convention for local artwork (png should also work). Artist thumb: artist.jpg; Album thumb: folder.jpg or cover.jpg; Fan Art (used as background in banners): fanart.jpg. Artist thumb and Fanart should be in the folder with the artist name. Album thumbs should be in the folder with the album name or in the disc folders below that. More about artwork file types can be found here https://kodi.wiki/view/Artwork_types
 - Embedded album thumbs will be extracted from audio files. However, you can improve performance and save disk space by providing a single local artwork file vs. embedding the same artwork in all files.
 
-!!! tip "Local Artwork is Optimal" Using embedded images impacts performance as the same query is called mutiple times per album. This is very noticeable with large numbers of files.
+
+!!! tip "Local Artwork is Optimal" 
+    Using embedded images impacts performance as the same query is called mutiple times per album. This is very noticeable with large numbers of files.
```
Comment (marcelveldt) [inline docs/music-providers/filesystem.md:39]:
```suggestion
    Using embedded images on every track of the same album is suboptimal for both disk space and performance. Use a single folder.jpg in the album's folder instead.
```

File: docs/release.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Release Cycle
+
+## Stable Releases
+
+Stable releases will only be pushed every 1-3 months, depending on the amount of features and quality. This stable release only comes after a beta cycle has been promoted to RC and the dev team all agree its stable enough to be promoted.
```
Comment (marcelveldt) [inline docs/release.md:5]:
```suggestion
Stable releases will only be published every 1-3 months, depending on the amount of features and quality. This stable release only comes after a beta cycle has been promoted to"Release Candidate" and the development team all agree its stable enough to be promoted.
```

File: docs/release.md
Comment (marcelveldt) [inline docs/release.md:1]: can we publish this only after/on may 9th ?

File: docs/release.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+Once 2.0 is released, the beta of 2.1 will be started which starts with the same code base as 2.0 stable. Users can decide if they want to stay on the beta channel.
+
+There is also a dev/nightly add-on which can be run to get the latest and greatest.
+
+It is possible to run the stable, beta, dev server add-on side by side if desired as they don't share any data. Thus it is possible to temporarily run the beta add-on to try out new features and then stop it again.
+
+## HA Integration
+
+For the HA integration there will also be stable and beta releases if there is a need. For example, if there are any breaking changes (or new features) in the beta add-on, that will be a reason to also have a beta release of the HA integration. Folks running the MA server on the beta channel are advised to also run the HA integration in beta to prevent compatibility issues and vice versa.
+
```
Comment (marcelveldt) [inline docs/release.md:22]: this text needs some love

File: docs/release.md
Comment (OzGav) [reply docs/release.md:1]: I could push another change to take this out and put it back in later. All the image updates would be handy now...but not critical?

File: docs/release.md (hunk 4 above)
Comment (OzGav) [reply docs/release.md:22]: Ill go through it again.

Comment (OzGav) [reply docs/release.md]: I have pushed another version

**Review by marcelveldt — APPROVED** [review]
Thanks!

**Pull request merged**